build.sbt

```diff
@@ -1,9 +1,74 @@
+import scala.collection.immutable.Seq
+
 lazy val `chatops4s` = (project in file("."))
   .settings(commonSettings)
+  .settings(
+    publish / skip := true
+  )
   .aggregate(
+    `chatops4s-core`,
+    `chatops4s-slack`,
+    `chatops4s-examples`,
     `chatops4s-discord`,
     `chatops4s-discord-example`,
   )
+lazy val `chatops4s-core` = (project in file("modules/chatops4s-core"))
+  .settings(commonSettings)
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.typelevel" %% "cats-effect" % "3.6.2",
+      "org.typelevel" %% "cats-core" % "2.13.0"
+    ),
+    Test / parallelExecution := false
+  )
+
+lazy val `chatops4s-slack` = (project in file("modules/chatops4s-slack"))
+  .settings(commonSettings)
+  .settings(
+    libraryDependencies ++= Seq(
+      "com.softwaremill.sttp.client4" %% "core" % "4.0.9",
+      "com.softwaremill.sttp.client4" %% "cats" % "4.0.9",
+      "com.softwaremill.sttp.client4" %% "circe" % "4.0.9",
+      "com.softwaremill.sttp.client4" %% "httpclient-backend-cats" % "4.0.9",
+      "com.softwaremill.sttp.tapir" %% "tapir-core" % "1.11.36",
+      "com.softwaremill.sttp.tapir" %% "tapir-cats" % "1.11.36",
+      "com.softwaremill.sttp.tapir" %% "tapir-http4s-server" % "1.11.36",
```

Comment:
As I mentioned before, lib module cannot have tapir interpreter dependencies. Interpretation should happen on user side (e.g. example project)

This also means all other dpens in this module, the goal is to have asbolute minimum of dependencies here.